core/controllers/email_dashboard.py

```diff
@@ -94,7 +95,7 @@ def post(self):
         self._validate(kwargs)
 
         user_query_id = user_query_services.save_new_user_query(
-            self.user_id, **kwargs)
+            self.user_id, kwargs)
```

Comment:
Why is this changed?

Reply:
In the save_new_user_query, I had to change to remove the mentions of individual predicate, so that when an additional predicate is added, less changes are required in the backend.